ontology/dprod/dprod-ontology.ttl

```diff
@@ -1,160 +1,95 @@
+  dct:publisher <https://www.omg.org/> ;
+  dct:contributor <https://www.linkedin.com/in/natasavaritimou/> ;
+  dct:contributor <https://www.linkedin.com/in/peterivett/> ;
+  dct:contributor <https://www.linkedin.com/in/jgeluk/> ;
+  dct:contributor <https://www.linkedin.com/in/marcel-fr%C3%B6hlich-648a043/> ;
+  dct:contributor <https://www.linkedin.com/in/matthias-jurisch/> ;
+  dct:contributor <https://www.linkedin.com/in/tonyseale/> ;
   owl:imports <http://www.w3.org/ns/dcat#> ;
-  owl:versionInfo "Create the Data Product Ontology based on input and output DCAT datasets"@en ;
-  foaf:maker
-    [
-      foaf:homepage <http://www.ekgf.org/> ;
-      foaf:name "Enterprise Knowledge Graph Foundation" ;
-    ] ;
+  owl:versionInfo "OMG Request For Comments Errata 2" ;
 .
 
-
 ############################# DPROD Classes  #########################
 
 dprod:DataProduct
   a owl:Class, rdfs:Class ;
-  dc:description
-    "A data product is a rational, managed, and governed collection of data, with purpose, value and ownership, meeting consumer needs over a planned life-cycle."@en ;
+  dct:description
+    "A rational, managed, and governed collection of data, with purpose, value and ownership, meeting consumer needs over a planned life-cycle."@en ;
   rdfs:comment "A data product may have input and output ports, code and metadata"@en ;
   rdfs:isDefinedBy dprod: ;
-  rdfs:label "Data Product" ;
+  rdfs:label "data product" ;
   rdfs:subClassOf dcat:Resource ;
 .
 
 dprod:DataProductLifecycleStatus
   a owl:Class, rdfs:Class ;
-  dc:description
-    "The lifecycle status of the data product taken from a control list ( Ideation, Design, Build, Deploy, Consume )."@en ;
-  rdfs:comment "The lifecycle of the data product as defined from EDM Council CDMC"@en ;
+  dct:description
+    "The development status of the data product taken from a controlled list ( Ideation, Design, Build, Deploy, Consume )."@en ;
+  rdfs:comment "The lifecycle of the data product as defined by EDM Council CDMC"@en ;
   rdfs:isDefinedBy dprod: ;
-  rdfs:label "Data Product Lifecycle Status" ;
+  rdfs:label "data product lifecycle status" ;
   rdfs:subClassOf dprod:Enumeration ;
+  rdfs:seeAlso <https://edmcouncil.org/wp-content/uploads/2024/03/EDMWebinar_DPSIG_04-30-Presentation.pdf#page=7> ;
 .
 
 dprod:InformationSensitivityClassification
   a owl:Class, rdfs:Class ;
-  rdfs:label "Information Sensitivity Classification" ;
-  dc:description
-    "A taxonomy of concepts that classifies the information within a dataset that indicates the level of control and protection that must be applied to the dataset."@en ;
+  rdfs:label "information sensitivity classification" ;
+  dct:description
+    "A classification of the information within a dataset to indicate the level of control and protection that must be applied."@en ;
   rdfs:isDefinedBy dprod: ;
   rdfs:seeAlso <https://spec.edmcouncil.org/cdmc/ontology/Classification/InformationSensitivity> ;
 .
 
 dprod:Enumeration
```

Comment:
This class is not included in the generated spec, shouldn't we?

Reply:
I don't know why it's not included. - that's a generator issue isn't it?

Reply:
Yes, will add it then